.profile.d/heroku-metrics-daemon.sh

```diff
@@ -32,13 +32,13 @@ echo "agentmon setup took ${ELAPSEDTIME} seconds"
 
 AGENTMON_FLAGS=()
 
-if [ -f pom.xml ] || [ -f build.gradle ] || [ -d .jdk ]; then
-    export JAVA_TOOL_OPTIONS="-javaagent:bin/heroku-metrics-agent.jar ${JAVA_TOOL_OPTIONS}"
-    AGENTMON_FLAGS+=("-prom-url=http://localhost:${HEROKU_METRICS_PROM_PORT}${HEROKU_METRICS_PROM_ENDPOINT}")
-elif [ -f build.sbt ]; then
+if [[ -f build.sbt ]]; then
     unzip -qq bin/heroku-metrics-agent.jar 'javax/*' -d bin/ext/
     export JAVA_OPTS="-javaagent:bin/heroku-metrics-agent.jar=cp=/app/bin/ext/ -Xbootclasspath/a:bin/heroku-metrics-agent.jar ${JAVA_OPTS}"
```

Comment:
yea, it's weird but correct. The first `=` indicates the beginning of the args to the agent. So logically it's `[heroku-metrics-agent.jar]=[[cp=/app/bin/ext/],...]`